utils/docker/Dockerfile.ubuntu

```diff
@@ -10,24 +11,22 @@
 ARG BASE_DISTRO=ubuntu:22.04
 FROM $BASE_DISTRO
 LABEL maintainer="daos@daos.groups.io"
+# Needed for later use of BASE_DISTRO
+ARG BASE_DISTRO
```

Comment:
Why is this needed if we have this above?
```
ARG BASE_DISTRO=ubuntu:22.04
```

Reply:
It is a special feature of Dockerfiles.
Any ARG statements before the FROM statement are invisible unless they are repeated without the optional initializer.

So this is the only way to pass the BASE_DISTRO through to the rest of the Dockerfile to handle version specific behavior.